packages/contracts/src/contract-dumps.ts

```diff
@@ -115,7 +115,7 @@ export const makeStateDump = async (cfg: RollupDeployConfig): Promise<any> => {
     deploymentSigner: signer,
     ovmGasMeteringConfig: {
       minTransactionGasLimit: 0,
-      maxTransactionGasLimit: 9_000_000,
+      maxTransactionGasLimit: 10_000_000,
```

Comment:
Since we've upped the `maxTransactionGasLimit` in EM we might want to up the `L2_GAS_DISCOUNT_DIVISOR` as well as now the burned gas will be higher for txs sent with a default gas limit. Or at least we could make a note to consider this as part of the work in #762

Reply:
Is it ok if we handle that in #762?